packages/fail-logs/hooks/capture-failures.py

```
@@ -90,6 +100,33 @@ def _resolve_project_dir(data: dict) -> str:
+    """記録時点の Git ブランチ名を解決する。取得失敗時は空文字列を返す(fail-safe)。"""
+    if timeout <= 0:
+        return ""
+
+    git_env = sanitized_git_env()
+    commands = (
+        ["git", "symbolic-ref", "--short", "HEAD"],
+        ["git", "rev-parse", "--abbrev-ref", "HEAD"],
+    )
+    for command in commands:
+        try:
+            result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)